Directory.Build.props

```diff
@@ -14,9 +14,6 @@
     <IsTestAssetProject Condition="$(RepoRelativeProjectDir.Contains('testassets'))">true</IsTestAssetProject>
     <IsSampleProject Condition="$(RepoRelativeProjectDir.Contains('samples'))">true</IsSampleProject>
 
-    <IsReferenceAssemblyProject>false</IsReferenceAssemblyProject>
```

Comment:
@dotnet/aspnet-build I wanted to double check removing reference assembly related logic here. Since everything left is package only, I think we can remove all ref assemblies logic right?

Reply:
Makes sense to me